src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -8764,7 +8764,7 @@ namespace SurfaceGeometry {
             }
 
             {
-                auto const SELECT_CASE_var(state.dataIPShortCut->cAlphaArgs(2));
+                std::string const &SELECT_CASE_var = state.dataIPShortCut->cAlphaArgs(2);
```

Comment:
> Ok to use `std::string const &` here because you are referencing `cAlphaArgs` storage rather than capturing a new `std::string` that is created by `MakeUPPERCase`.

`getObjectItem` pushes all alphas to UPPERCase [here]() so there's never a need to reapply `MakeUPPERCase` to `cAlphaArgs` (or equivalent). The exception to this are fields tagged with \retaincase.

Reply:
Thanks. I noted that. I will do a sweep for that in a near-future PR. Is the same true for the epJSON `x.get<std::string>()` path?

Reply:
> Thanks. I noted that. I will do a sweep for that in a near-future PR. Is the same true for the epJSON `x.get<std::string>()` path?

No, raw epJSON returns the string directly from the input file. `getAlphaFieldValue` returns strings in UPPERcase, because it calls `getObjectItemValue` (which also returns the default value if the field is empty). All of these functions could use some comments to note these things.